src/twisted/conch/scripts/ckeygen.py

```diff
@@ -220,7 +245,7 @@ def _saveKey(key, options):
     @param options:
     @type options: L{dict}
     """
-    keyTypeName = key.type().lower()
+    keyTypeName = 'ecdsa' if key.type() == 'EC' else key.type().lower()
```

Comment:
I'm with @moshez here: let's have an explicit dictionary registry that handles this mapping.